app/src/main/java/com/ucmobiledevelopment/freeways/dto/IncidentDTO.kt

```diff
@@ -2,15 +2,15 @@ package com.ucmobiledevelopment.freeways.dto
 
 data class IncidentDTO(
     val CITY: String? = "",
-    val CITYNAME: String? = "",
+    val CITY_NAME: String? = "",
     val COUNTY: String? = "",
-    val COUNTYNAME: String? = "",
-    val CaseYear: String? = "",
+    val COUNTY_NAME: String? = "",
+    val CASE_YEAR: String? = "",
     val FATALS: String? = "",
     val LATITUDE: String? = "",
     val LONGITUDE: String? = "",
     val STATE: String? = "",
-    val STATENAME: String? = "",
+    val STATE_NAME: String? = "",
```

Comment:
I changed these to meet this requirement of Kotlin naming:

![immutable](https://user-images.githubusercontent.com/60935069/156949321-bf732fca-c3aa-480d-b55a-78fd2fca4584.PNG)

From my understanding of this data class, this data will end up fulfilling the requirements of this rename.